### ISSUE TITLE
GL.Fixed.Lighting fails to compile under GNAT Community 2019

### ISSUE THREAD

**Issue by rogermc2** [flyx/OpenGLAda#136]
I have installed GNAT Community 2019.
However, `GL.Fixed.Lighting` compilation fails with:
~~~
gl-fixed-lighting.ads:75:13: warning: use of an anonymous access type allocator
~~~
This warning is occurs for each item in `Lights : constant array`.

**Comment by flyx** [comment]
This should be fixed with the merging of #126, right?

**Comment by flyx** [comment]
I just removed the whole code that caused this. It was idiotic and could be done without allocating anything on the heap.
